.github/workflows/e2e-testnet.yml

```diff
@@ -52,6 +52,6 @@ jobs:
         working-directory: e2e
         env:
           CHAIN_ENVIRONMENT: paseo-testnet
-          WS_PROVIDER_URL: wss://0.rpc.testnet.amplica.io
+          WS_PROVIDER_URL: ${{ vars.E2E_TESTNET_PASEO_RPC }}
```

Comment:
Variable added

This will allow us to target other RPC nodes if needed, without a code change.

Reply:
good idea.